lib/IRGen/GenClass.cpp

```diff
@@ -1062,6 +1064,20 @@ namespace {
       }
     }
 
+    /// Add the protocol to the vector, if it's ObjC,
+    /// or search it's superprotocols.
```

Comment:
Nit: `it's` -> `its` on this line only 😛
(Also maybe consider spelling out "Objective-C" here and below)